apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/TableEditor/ForeignKeysManagement/ForeignKeyRow.tsx

```diff
@@ -78,9 +78,10 @@ export const ForeignKeyRow = ({
               }
             >
               <EditorTablePageLink
```

Comment:
If we're opening in a new tab there's no reason to spend network resources on prefetching table data... can you please swap this for a regular next/link? 🙏

Reply:
oh yes! that makes sense haha